stories/features/rewards/wallet.tsx

```diff
@@ -84,10 +92,45 @@ storiesOf('Feature Components/Rewards/Wallet', module)
       </div>
     )
   })
+  .add('Panel', () => {
+    return (
+      <div style={{ width: '373px', background: '#f9fbfc' }}>
+        <WalletPanel
+          id={'wallet-panel'}
```

Comment:
Please add options (knobs) for this component, so that user can interact with it

Reply:
++